tests/integration/targets/rds_cluster/roles/rds_cluster/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+- name: rds_cluster integration tests
+  module_defaults:
+    group/aws:
+      region: '{{ aws_region }}'
+      aws_access_key: '{{ aws_access_key }}'
+      aws_secret_key: '{{ aws_secret_key }}'
+      security_token: '{{ security_token | default(omit) }}'
+
+  collections:
+  - community.aws
```

Comment:
it seems that this dependency can be removed, as all modules involved in this test are part of the `amazon.aws` collection, same as next test

Reply:
Removed, thanks!